src/services/expense.service.js

```diff
@@ -0,0 +1,80 @@
+const { Op } = require('sequelize');
+const { Expense } = require('../models/Expense.model.js');
+
+const getExpenses = async ({ userId, categoryList, from, to }) => {
+  const whereClause = {};
+
+  if (userId) {
+    whereClause.userId = userId;
+  }
+
+  if (categoryList && categoryList.length > 0) {
+    whereClause.category = { [Op.in]: categoryList };
+  }
+
+  if (from) {
+    whereClause.spentAt = { [Op.gte]: new Date(from) };
+  }
+
+  if (to) {
+    whereClause.spentAt = { ...whereClause.spentAt, [Op.lte]: new Date(to) };
+  }
+
+  const expenses = await Expense.findAll({
+    where: whereClause,
+  });
+
+  return expenses;
+};
+
+const createExpense = ({
+  userId,
+  spentAt,
+  title,
+  amount,
+  category = '',
+  note = '',
```

Comment:
The default values for `category` and `note` are empty strings. Ensure that this aligns with your application's requirements, especially since the model requires these fields to be non-null.